testing/sndio/sndio.initd

```diff
@@ -0,0 +1,27 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+command=/usr/bin/sndiod
+command_args="$sndio_args"
+name="sndio sound daemon"
+
+depends() {
+	need localmount
+	use net
+}
+
+start() {
```

Comment:
Is there a reason you need to define custom start/stop functions?

The openrc [styleguide][] advises to not define your own.

[styleguide]:https://github.com/OpenRC/openrc/blob/master/service-script-guide.md#dont-write-your-own-startstop-functions

Reply:
nope i just copy pasted a sndio from somewhere

Reply:
This service file doesn't even seem to work. It looks like sndiod does not want to run as root ("sndiod: setpriority: Permission denied"). If I run it as a normal user, it just seems to work.